WordPress/Sniffs/WP/I18nSniff.php

```diff
@@ -80,6 +80,29 @@ class I18nSniff extends AbstractFunctionRestrictionsSniff {
 		[bcdeEfFgGosuxX]           # Type specifier.
 	)/x';
 
+	/**
+	 * Match translatable strings wrapped in HTML, e.g. '<address>123 Fake Street</address>'
+	 */
+	const WRAPPED_IN_HTML_REGEX = '/(?:
```

Comment:
Thanks! This is quite insightful, and I'll keep it handy as a reference for the future :slightly_smiling_face: 

For now, I've decided to use an actual HTML Parser rather than RegExps, see https://github.com/WordPress/WordPress-Coding-Standards/pull/1856#issuecomment-581280385.